### PR TITLE
Add timestamp support back

### PR DESCRIPTION
**Pull request by caglar10ur**

### CODE REVIEW

**Comment by anchal-agrawal** [comment]
lgtm

File: pkg/trace/trace.go [hunk 1]
```diff
@@ -27,9 +27,14 @@ var Logger = &log.Logger{
...
-	Hooks:     make(log.LevelHooks),
-	Level:     log.InfoLevel,
+	Formatter: &log.TextFormatter{
+		ForceColors:   true,
+		FullTimestamp: true,
+		// from https://godoc.org/github.com/docker/docker/pkg/jsonlog#pkg-constants
+		TimestampFormat: "2006-01-02T15:04:05.000000000Z07:00",
+	},
+	Hooks: make(log.LevelHooks),
+	Level: log.InfoLevel,
```
Comment (hmahmood) [inline pkg/trace/trace.go]: In my PR I have changed this to use the standard logger, so this change may not be necessary.

Comment (fdawg4l) [inline pkg/trace/trace.go]: @caglar10ur is right; it was always INFO.

**Comment by fdawg4l** [comment]
LGTM

**Comment by caglar10ur** [comment]
OK, I think @fdawg4l and @hmahmood  are in the same office today and I'm not sure what will be the effect of going back to regular logger. 

Could you go over the other PR and decide whether we need  this or not? If you think this is no longer needed please feel free to close.

**Comment by caglar10ur** [comment]
Closing this as I believe this is implemented differently

**Pull request closed without merging**